NostalgiaForInfinityX.py

```diff
@@ -8848,65 +8861,65 @@ def custom_sell(self, pair: str, trade: 'Trade', current_time: 'datetime', curre
         if all(c in ['31', '32', '33', '34', '35', '36'] for c in buy_tags):
```

Comment:
This will not be met in live/dry with the extra version tag in the list.

Reply:
fixed